docs/workflow/building/libraries/README.md

```diff
@@ -77,9 +77,9 @@ The libraries build has two logical components, the native build which produces
 
 The build configurations are generally defaulted based on where you are building (i.e. which OS or which architecture) but we have a few shortcuts for the individual properties that can be passed to the build scripts:
 
-- `-framework|-f` identifies the target framework for the build. It defaults to latest `netcoreapp` but possible values include `netcoreapp` (Which will map to latest netcoreapp) or `net472`. (msbuild property `TargetGroup`)
+- `-framework|-f` identifies the target framework for the build. It defaults to latest `netcoreapp` but possible values include `netcoreapp` (Which will map to latest netcoreapp) or `net472`. (msbuild property `BuildTargetFramework`)
```

Comment:
"Which will map to latest netcoreapp". This isn't true anymore. Please update that section and mention that currently netcoreapp5.0 needs to be passed in.